### PR TITLE
Check if exif exists before getting it

### PR DESCRIPTION
**Pull request by joetempleman**

### CODE REVIEW

**Comment by cwsullivan** [comment]
LGTM, why is this failing?

File: opencv_engine/engine.py [hunk 1]
```diff
@@ -286,3 +288,14 @@ def flip_horizontally(self):
...
+    def _get_exif_segment(self):
+        """ Override because the superclass doesn't check for no exif.
+        """
+        segment = None
+        try:
+            if getattr(self, 'exif', None) is not None:
+                segment = ExifSegment(None, None, self.exif, 'ro')
+        except Exception:
+            logger.warning('Ignored error handling exif for reorientation', exc_info=True)
+        return segment
```
Comment (cwsullivan) [inline opencv_engine/engine.py]: Is this better style than

`""" docstring"""
         try:
             if getattr(self, 'exif', None) is not None:
                 return ExifSegment(None, None, self.exif, 'ro')
         except Exception:
             logger.warning('Ignored error handling exif for reorientation', exc_info=True)
             return None

Comment (joetempleman) [inline opencv_engine/engine.py]: could do it either way but I prefer the explicit returns

**Comment by joetempleman** [comment]
@cwsullivan failing because there are no tests but circle latched on

**Pull request merged**